src/js/movingDots.js

```diff
@@ -0,0 +1,24 @@
+import $ from 'jquery';
+
+export default function dotsMovement() {
+  $(document).on('mousemove', (event) => {
+    $('.l-dot').each((i, dot) => {
+      const dotsNumber = $('.l-dot').length;
+      let modifier;
+      if (i % 2) {
+        modifier = i;
+      } else {
+        modifier = 3 * (0.5 * dotsNumber - i);
+      }
```

Comment:
LGTM, but I would try to avoid `let`. So `const modifier = i % 2 ? i : 3 * (0.5 * dotsNumber - i);` or maybe `const modifier = computeModifier(i, dotsNumber);`